docs/troubleshoot/cli/mustgather-cli.md

```diff
@@ -9,24 +9,24 @@ Follow these instructions to gather specific pieces of information to help troub
 Issue the following command:
 
 ```
-zowe -V 
-``` 
+zowe -V
+```
 
-Zowe CLI versions may vary depending upon if the `@latest` or `@lts-incremental` version is installed.   
+Zowe CLI versions may vary depending upon if the `@latest` or `@zowe-v1-lts` version is installed.
 
 For the `@latest` (forward-development) version:
 
 ```
 npm list -g @zowe/cli
 ```
 
-For the `@lts-incremental` version:
+For the `@zowe-v1-lts` version:
```

Comment:
The same command is issued for both zowe-v1-lts and latest. Maybe group them together and restore the blob for lts-incremental?